setup.py

```diff
@@ -39,8 +39,7 @@
 
 install_requires = [
     "click>=7",
-    "cwltool==1.0.20191022103248",
-    "cwl-utils==0.5",
```

Comment:
Testing with:

```console
$ reana-dev git-status -c ALL | grep -v '@ master'
reana-client @ pr-538 @ fd3db82 cwltool: upgrade to the latest version
reana-workflow-engine-cwl @ pr-192 @ 0bf38f4 cwltool: upgrade to the latest version
```

I'm getting installation trouble if `cwl-utils` is not pinned anymore and if I upgrade current venv:

```console
$ reana-dev client-install    
...
[2021-08-30T10:49:23] reana: pip check
cwl-utils 0.5 has requirement schema-salad<7, but you have schema-salad 8.1.20210721123742.
[2021-08-30T10:49:23] reana: Command 'pip check' returned non-zero exit status 1.
```

Perhaps we could install some at least greater-then lower-than boundaries for packages, for people upgrading from a previously unknown state?

Note that installing into an empty venv is working perfectly fine... So this is a nice-to-have, not a should-have remark...

Reply:
Adding `"cwl-utils>=0.10",` solves the issue, but the downside of this approach is that we will have a requirement of unused dependency

Reply:
Hm, actually it brings other issues, [tests are not passing](https://github.com/reanahub/reana-client/runs/3461008382?check_suite_focus=true#step:6:172) now because of the typing errors. `cwl-utils` seems to bring `typing` module which causes issues with `cwltool`. Perhaps it would be better to get rid of it?

Reply:
OK!